src/powerman/powerman.c

```diff
@@ -222,6 +225,12 @@ int main(int argc, char **argv)
         case 'I':              /* --ignore-errs */
             ignore_errs = true;
             break;
+        case 'R':              /* --retry-connect=N (sleep 1s after fail) */
+            errno = 0;
+            retry_connect = strtoul (optarg, NULL, 10);
+            if (errno != 0 || retry_connect < 1)
+                err_exit(false, "could not parse --retry-connect value");
```

Comment:
if retry_connect=0, it isn't a parse error?  Maybe you mean "invalid input to ..."?